### PR TITLE
iiif, updated states for loris containerisation

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

**Comment by lsh-0** [comment]
not sure how we'll roll this out without affecting prod.

perhaps target iiif ci/end2end/continuumtest environments first?

**Comment by giorgiosironi** [comment]
Yes, ideally it should all be working on `end2end` first. You can run https://alfred.elifesciences.org/job/test-elife-spectrum/ after applying it there to smoke test it.

**Comment by giorgiosironi** [comment]
Also, the servers are load balanced in all environments so you can:

- turn off `nginx` on server 1 to make the load balancer detach it
- apply the change which will probably turn on `nginx` if everything succeeds
- run further smoke tests if provided

If anything goes wrong, keeping `nginx` down should avoid traffic going to the broken server. There's a Fastly cache on top anyway, but if a wrong image gets out (with 200 OK) it's very difficult to replace.

**Comment by lsh-0** [comment]
> but if a wrong image gets out (with 200 OK) it's very difficult to replace.

This is good to know. I haven't seen any spurious 200 responses yet.

I'll tweak this PR slightly so that `prod` is protected from any changes, then update end2end first, trigger a build and go from there.

File: salt/top.sls [hunk 1]
```diff
@@ -392,9 +392,17 @@ base:
         - elife.uwsgi
         - elife.external-volume
         - elife.newrelic-python
-        - iiif
+
+    'iiif--prod--*':
         - iiif.loris
 
+    # all iiif instances *except* prod
+    'iiif--* and not iiif--prod--*':
```
Comment (lsh-0) [inline salt/top.sls]:
```
# salt -C 'iiif--* and not iiif--prod--*' --preview-target
- iiif--end2end--1
- iiif--continuumtest--2
- iiif--pr-20-fresh--1
- iiif--pr-20-base-update--1
- iiif--ci--1
- iiif--containerised1--1
- iiif--pr-21-fresh--1
- iiif--pr-22-fresh--1
- iiif--pr-18-base-update--1
- iiif--ci--4
- iiif--containerised3--1
- iiif--ci--3
- iiif--pr-18-fresh--1
- iiif--pr-22-base-update--1
- iiif--containerised2--1
- iiif--pr-21-base-update--1
- iiif--continuumtest--1
- iiif--end2end--2
- iiif--ci--2
```

**Pull request merged**